dist/s_clang-format.list

```diff
@@ -0,0 +1,2 @@
+src/support/hash_city.c
+src/support/hash_fnv.c
```

Comment:
The file is missing a terminating `newline`.

Reply:
8493ae7
Fixed. Also added a few more third party files that I saw listed here:
https://source.wiredtiger.com/3.1.0/license.html